Watchman.Discord/WatchmanBot.cs

```diff
@@ -105,6 +105,8 @@ public WorkflowBuilder GetWorkflowBuilder()
                         .AddHandler(this.PrintDebugExceptionInfo, onlyOnDebug: true)
                         .AddHandler(this.PrintExceptionOnConsole);
                 })
+                .AppendOnUserAddedReaction(reactionContext => _context.Resolve<ReactionsService>().UserAddedReaction(reactionContext))
```

Comment:
czemu zmuszamy użytkownika frameworka do dodawania tych linijek? to powinno dziać sie w środku frameworka

Reply:
Najpierw wy chcieliście cały ten mechanizm przenieść do Watchman.Discord a teraz ciągle przy review karzecie mi to znowu umieścić w frameworku... To tylko jedna linijka, w której przekazujemy obiekt serwisu i jego delegat, więc moim zdaniem nie jest to jakieś wielkie obciążenie dla usera skoro to jest część konfiguracji a resztę robi framework.

Reply:
powinno to wyglądać tak:
```csharp
.AddOnUserAddedReaction(builder =>
{  
    builder  
        .AddFromIoC<ReactionsService>(x => x.UserAddedReaction)  
})
```

Reply:
`nie jest to jakieś wielkie obciążenie dla usera skoro to jest część konfiguracji a resztę robi framework.`

jak w jednym przypadku będziemy kazali użytkownikowi robić coś co powinno dziać sie automatycznie
to po chwili znajdzie sie więcej przypadków gdzie ktoś zrobi coś `bo przecież tam mamy to tak zrobione`

lepiej tego unikać
pewnie będą sytuacje gdzie tego nie unikniemy (nie wykluczam ich)
ale jak da sie uniknąć, to unikajmy